src/puptoo/utils/puptoo_logging.py

```diff
@@ -49,3 +54,24 @@ def initialize_logging():
         logger.info("Skipping cloudwatch configuration")
 
     return logger
+
+
+class ContextualFilter(logging.Filter):
+    """
+    This filter gets the request_id from the message and adds it to
+    each log record. This way we do not have to explicitly retreive/pass
+    around the request_id for each log message
```

Comment:
this is a cool way to do it! I always had to pass request_id around and it was messy.